dashboard/app/models/game.rb

```diff
@@ -227,6 +227,7 @@ def self.setup
         ScriptCompletion:script_completion
         StandaloneVideo:standalone_video
         ExternalLink:external_link
+        EvaluationQuestion:evaluation_question
```

Comment:
I wanted this to be distinct from existing multi types - especially since there's no possible of picking multiple answers for this.

That being said, the fact that this inherits from Multi is a compelling point